libs/pages/elewa/about-us/src/lib/elewa-about-us.module.ts

```diff
@@ -21,6 +21,9 @@ import { NextDirective } from './directives/next.directive';
 
 import { PrevDirective } from './directives/prev.directive';
 import { AboutUsRoutingModule } from './about-us.routing';
+import { AboutUsWeCareComponent } from './components/about-us-we-care/about-us-we-care.component';
+import { ElewaVerticalIconAndTextComponent } from 'libs/features/components/cards/src/lib/cards/elewa-vertical-icon-and-text/elewa-vertical-icon-and-text.component';
```

Comment:
Import from the aliased path, In your case you need to import the module that exports this component.